storage-resize-images/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Version 0.2.0
+
+fixed - Resized images now render in the Firebase console. (Issue #140)
+fixed - Fixed bug that was generating the wrong image when another image was
+uploaded with the same name. (Issue #286)
```

Comment:
I edited this 2nd bullet in this release note.  Can we pls get verification that this is an accurate description of what was fixed?

Reply:
Hmm, looks like the title of the fix is a little clearer - would you prefer that? `clear cache so that image names can be reused` https://github.com/firebase/extensions/pull/297

Reply:
Determined a clearer release note msg offline.